src/app/components/core/configuration-manager/view-config-item/view-config-item.component.html

```diff
@@ -1,31 +1,55 @@
 <div class="columns is-mobile is-multiline" *ngFor="let categoryData of categoryConfigurationData?.value">
-    <div class="column is-narrow is-half" *ngFor="let configItem of (categoryData | keys)">
```

Comment:
we shall need to ensure, add service and add north task wizard are good with it.